docs/configuration/images.md

```diff
@@ -496,4 +496,4 @@ prefixed with `argocd-image-updater.argoproj.io`.
 |`write-back-method`|The method used for writing back updates. Either can be `argocd` (imperative) or `git` (declarative)|
 |`write-back-target`|The target used for writing back updates. Either can be `kustomization` (For kustomize) or `helmvalues` (For helm)| 
 |`git-branch`|Specify the branch in which updates will be wrote to|
-|`git-repository`|A URL to a git repository. If provided will override the RepoURL that is provided to the ArgoCD application|
\ No newline at end of file
+|`git-repository`|A URL to a git repository. If provided will override the RepoURL that is provided to the ArgoCD application|
```

Comment:
I don't see any diference between teh 2 lines.

Reply:
thank you for your comment!

Reply:
redundant change with Vim. Fixed it, thank you.